easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20130501-R1_openblas.patch

```diff
@@ -0,0 +1,45 @@
+--- config.orig	2015-03-03 10:45:24.562802000 +0100
```

Comment:
I think we should make it a habit of always adding a line of comment on top of a patch about what it does